### PR TITLE
Funding plugin updates for 3.3 and 3.4

### PR DESCRIPTION
**Pull request by ajnyga**

### CODE REVIEW

**Comment by jonasraoni** [comment]
@bozana / @asmecher

File: plugins.xml [hunk 1]
```diff
@@ -2640,6 +2640,20 @@
 			<certification type="reviewed"/>
 			<description>This release adds support for OPS 3.3.0. It also adds translation for Brazilian Portuguese</description>
 		</release>
+		<release date="2024-11-16" version="3.3.0.3" md5="bbf81f9c47bc77a9b90881b887e5d00a">
+			<package>https://github.com/ajnyga/funding/releases/download/3.3.0.3/funding.tar.gz</package>
+			<compatibility application="ojs2">
+				<version>~3.3.0.0</version>
```
Comment (asmecher) [inline plugins.xml:2646]: @ajnyga, the `~` version syntax was only added with 3.3.0-11; that's why you see above entries with 3.3.0-0 to 3.3.0-10 listed explicitly. This entry won't work with those versions.

Comment (ajnyga) [reply plugins.xml:2646]: ahah, I was wondering what that was about. I will fix these

Comment (ajnyga) [reply plugins.xml]: @asmecher this should be fixed now

**Comment by asmecher** [comment]
@ajnyga, should I merge this? Or wait for the upcoming revision for the API changes?

**Comment by ajnyga** [comment]
please wait, you will have a new pr later today

**Comment by ajnyga** [comment]
Closed this and added a new pr https://github.com/pkp/plugin-gallery/pull/338

**Pull request closed without merging**